python/conda/libastra/build.sh

```diff
@@ -1,15 +1,26 @@
-cd build/linux
-./autogen.sh
-./configure --with-cuda=$CUDA_ROOT --prefix=$PREFIX
-if [ $MAKEOPTS == '<UNDEFINED>' ]
-  then
-    MAKEOPTS=""
+#!/bin/sh
+
+cd $SRC_DIR/build/linux
+
+$SRC_DIR/build/linux/autogen.sh
+
+# Add C++11 to compiler flags if nvcc supports it, mostly to work around a boost bug
+NVCC=$CUDA_ROOT/bin/nvcc
+echo "int main(){return 0;}" > $CONDA_PREFIX/test.cu
+$NVCC $CONDA_PREFIX/test.cu -ccbin $CC --std=c++11 -o $CONDA_PREFIX/test.out > /dev/null
```

Comment:
The idea is to use the exit code of nvcc to decide whether or not to use the flag. But my bash skills are bad enough not to understand properly what's going on.
So here's a little test:
```bash
#!/bin/sh

NVCC=/usr/local/cuda/bin/nvcc
echo "int main(){return 0;}" > test__.cu
$NVCC --madeup-flag test.cu -o test__
if [ $? = 0 ]; then
    echo "success"
else
    echo "failed but still alive"
fi
rm -f test__
echo "made it to the end"
```
The output is:
```
nvcc fatal   : Unknown option '-madeup-flag'
failed but still alive
made it to the end
```
And the whole program exits with code 0:
```bash
$ echo $?
0
```